cmd/buildah/images.go

```diff
@@ -51,11 +61,26 @@ func imagesCmd(c *cli.Context) error {
 	if c.IsSet("notruncate") {
 		truncate = !c.Bool("notruncate")
 	}
+	JsonOut := false
+	if c.IsSet("json") {
+		JsonOut = c.Bool("json")
+	}
 	images, err := store.Images()
 	if err != nil {
 		return errors.Wrapf(err, "error reading images")
 	}
-
+	if JsonOut {
```

Comment:
I agree with @runcom use this here.
```
if c.IsSet("json") {
```

And eliminate the code above.  If we are using this syntax elsewhere we should clean those up.